Bottom: dea61b38ea27d057426514c15564d471df671369
Top:    f0ed59e417aa191f61678a3ce546b86b32d859e5
Author: Robert Collins <robertc@robertcollins.net>
Date:   2013-03-06 22:35:10 +1300

Refresh of stream-to-dict

---

diff --git a/setup.py b/setup.py
index 4b9257d..b9a7677 100755
--- a/setup.py
+++ b/setup.py
@@ -70,6 +70,7 @@ setup(name='testtools',
       install_requires=[
         'extras',
         # 'mimeparse' has not been uploaded by the maintainer with Python3 compat
+        # but someone kindly uploaded a fixed version as 'python-mimeparse'.
         'python-mimeparse',
         ],
       )
